airgun/views/activationkey.py

```diff
@@ -82,6 +82,20 @@ class details(SatTab):
         name = EditableEntry(name='Name')
         description = EditableEntry(name='Description')
         hosts_limit = EditableLimitEntry(name='Host Limit')
+        host_limit_edit_btn = Text(
+            locator='//dd[@bst-edit-custom="activationKey.max_hosts"]//div[@ng-click="edit()"]'
+        )
+        unlimited_content_host_checkbox = Checkbox(
+            locator='//input[@ng-model="activationKey.unlimited_hosts"]'
+        )
+        host_limit_input = TextInput(locator='//input[@ng-model="activationKey.max_hosts"]')
+        host_limit_save_btn = Text(
+            locator='//dd[contains(@bst-edit-custom, "activationKey.max_hosts")]//button[@ng-click="save()"]'
+        )
+        host_limit_cancel_btn = Text(
+            locator='//dd[contains(@bst-edit-custom, "activationKey.max_hosts")]//button[@ng-click="cancel()"]'
+        )
```

Comment:
You can also modify host limit on the new AK page. Would it be appropriate to add support for that here?

Reply:
I think that new PR would be more appropriate for it